benchmark/sirun/exporting-pipeline/index.js

```diff
@@ -12,7 +12,10 @@ const config = {
   url: 'http://localhost:8126',
   flushInterval: 2000,
   flushMinSpans: 1000,
-  protocolVersion: process.env.ENCODER_VERSION
+  protocolVersion: process.env.ENCODER_VERSION,
+  stats: {
+    enabled: false
```

Comment:
I just added that because that benchmark crashes without it.

What should the benchmark be? Adapting this one to measure both with and without stats enabled? Or make a separate benchmark?

Reply:
I think this one should be adapted so that it's a variant here, yeah.